blockware/matrix/src/debug.cpp

```diff
@@ -0,0 +1,82 @@
+
+// void debug_alphabet()
```

Comment:
I'd either remove this file or disable it with an `#ifdef` so it's clear how to bring it back if someone needs to use it

Reply:
OR, just leave the uncommented and i _think_ the compiler is smart enough to remove these if they are never called.

Reply:
i was originally using these to understand the generated unifont but once i no longer had a need for them they just took up a ton of space. i didn’t wanna necessarily lose them so i commented them out and moved them here. if i un comment it fails to compile because they were meant to be included in main.cpp (at the time) now they would make sense in MatrixRain.cpp.   you’re right it’d be better to keep it compiled or remove it

Reply:
refactored things a bit and moved the alphabets into Alphabets class and used in both MatrixRain and DebugAlphabet, defining these functions and including in the main.cpp
by default i do think they are skipped but at least now you can uncomment the lines in main.cpp to run the debug demos